dev-packages/application-manager/src/generator/backend-generator.ts

```diff
@@ -91,7 +91,7 @@ const serverModule = require('./server');
 const serverAddress = main.start(serverModule());
 
 serverAddress.then(({ port, address }) => {
-    if (process && process.send) {
+    if (process.send) {
```

Comment:
This is what I mean by "drive by fix". Yes, it's a good fix, but unrelated to the problem at hand.

Reply:
Fair enough on this one.